.eleventy.js

```diff
@@ -11,6 +11,11 @@ module.exports = (eleventyConfig, attributes = {}) => {
 
     const globalAttributes = { ...defaultAttributes, ...attributes };
 
+    const phosphorCorePath = path.join(
+        path.dirname(require.resolve("@phosphor-icons/core")),
```

Comment:
Note: `require.resolve("@phosphor-icons/core")` resolves the entrypoint of that package, which is `<phosphor_path>/dist/index.umd.js`. So to get to the assets directory, we have to go up one folder first.